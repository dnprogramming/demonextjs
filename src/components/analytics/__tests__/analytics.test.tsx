import { render } from "@testing-library/react";
import Analytics from "../analytics";
import React from "react";

describe("Analytics Component", () => {
    it("renders analytics component unchanged", () => {
        const { container } = render(<Analytics />);
        expect(container).toMatchSnapshot();
    });
});