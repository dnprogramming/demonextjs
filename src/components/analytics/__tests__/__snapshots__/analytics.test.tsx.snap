// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Analytics Component renders analytics component unchanged 1`] = `<div />`;
